packages/anasazi/src/AnasaziTraceMinRitzOp.hpp

```diff
@@ -89,7 +105,7 @@ template <class Scalar, class MV, class OP>
 class TraceMinProjOp
 {
   typedef Anasazi::MultiVecTraits<Scalar,MV>    MVT;
-  const Scalar ONE; 
```

Comment:
I think a file or two must have slipped by with trailing whitespace stripped out.  It'll only be trailing whitespace that got changed though.

I used the checkin script to do the push per instructions in the [checkin-testing guide](https://github.com/trilinos/Trilinos/wiki/Policies-%7C-Safe-Checkin-Testing).  

The only test in the suite that gave me problems in testing was the MueLu test referenced above, that I had to disable.  

I'll work on reverting the file packages/anasazi/src/AnasaziTraceMinRitzOp.hpp so that only the header is changed.